src/main/java/de/tum/in/www1/exerciseapp/domain/MultipleChoiceSubmittedAnswer.java

```diff
@@ -76,7 +76,7 @@ public boolean isSelected(AnswerOption answerOption) {
      * Check if answerOptions were deleted and delete reference to in selectedOptions
      * @param question the changed question with the answerOptions
      */
-    public void checkForDeletedAnswerOptions(MultipleChoiceQuestion question) {
```

Comment:
same comment as above, please use a different method name of you actually delete something --> If I read "check", I would assume that nothing is deleted